setsofdata.py

```diff
@@ -63,14 +105,76 @@ def create_clinic_services(clinic_spec_data_frame):
             service_name_short_first = "".join([word[0] for word in service_name_first.split()]).upper() 
             service_name_short_second = "".join([word[0] for word in service_name_second.split()]).upper() 
 
-            all_services = [[service_name_first,service_name_short_first],[service_name_second,service_name_short_second]]
-            all_service_df = pd.DataFrame(all_services, columns=['ServiceName','ServiceShort'])
+            base_cost = 100
+            random_price_part = random.choice([100,120])
+
+            service_price_first = (base_cost + random_price_part) * 1
+            service_price_second = (base_cost + random_price_part) * 1.25
+         
+
+            service_cost_first = random_price_part * 1
+            service_cost_second = random_price_part * 1.25
+   
+            base_duration = 15
+
+            service_duration_first = base_duration
+            service_duration_second = base_duration
+
+            all_services = [\
+            [service_name_first,service_name_short_first,service_price_first,service_cost_first,service_duration_first]\
+            ,[service_name_second,service_name_short_second,service_price_second,service_cost_second,service_duration_second]]
+
```

Comment:
that's repeating for the third time in this conditional, and very closely resembles previous instances. maybe construct the parameters from function or class?